doc/components_1.md

```diff
@@ -32,6 +32,7 @@ A MJML document starts with a `<mjml>` tag, it can contain only `mj-head` and `m
 
 attribute | unit | description | default value
 ----------|------|-------------|---------------
+owa | string | if set to "desktop", will force desktop display on outlook. Otherwise outlook will display mobile version since it ignores media queries | none
```

Comment:
We should maybe update this "Switch displayed version for older (self-hosted) version of Outlook.com" And may be add a link to it #2241 ?